app/src/main/java/com/example/scoutconcordia/MapInfoClasses/CustomInfoWindow.java

```diff
@@ -1,13 +1,16 @@
-package com.example.scoutconcordia;
+package MapInfoClasses;
```

Comment:
should be: package com.example.scoutconcordia.MapInfoClasses;

Reply:
Fixed